src/app/Shared/Services/Api.service.tsx

```diff
@@ -259,8 +261,8 @@ export class ApiService {
           method: 'POST',
           body: form,
         }).pipe(
-          map((resp) => resp.ok),
-          catchError((_) => of(false)),
+          map((resp) => resp),
+          catchError((_) => of()),
```

Comment:
I think this will cause the chain of observable to hang and never complete. There are 2 ways we can go around this:

1. Change method return signature to `Observable<Response | undefined>` and return `undefined` in catchError?
2. Or looks like we status code from response to why not return `Observable<{ok: boolean, status: number}>`, and return `{}` in catchError?

What do you think?

Reply:
Use this to check httpError:

https://github.com/cryostatio/cryostat-web/blob/47b7c85cc20cb28e629e024f9bfd8f5dd0e91e90/src/app/Shared/Services/Api.service.tsx#L63

Reply:
Okay, I'm leaning towards 2, but I will work it out, thanks.